### PR TITLE
Custom apps kai usage

### PR DESCRIPTION
**Pull request by savitharaghunathan**

### CODE REVIEW

File: docs/customApps.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
+# Run Kai with your Applications
+
+Konveyor AI (KAI) leverages LLMs to streamline the modernization of application source code. To learn how Kai works, run this demo <insert demo link></insert>
```
Comment (savitharaghunathan) [inline docs/customApps.md:3]: Once the demo PR merges, I will update the link here

File: docs/customApps.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+```python
+ "<app name>": [
+        "https://github.com/<path to repo>",
+        "<source branch>",
+        "<migrated/target branch>",
+    ],
+```
+
+<app name>: The name of your application.
```
Comment (jwmatthews) [inline docs/customApps.md:27]: I don't think the '<app name>:' and '<path to repo>:' syntax rendered as you wanted.  

Screenshot 2024-05-21 at 2 19 11 PM

**Review by jwmatthews — APPROVED** [review]
LGTM, just assume you want a small syntax tweak for the 2 lines that are not rendering as expected

File: docs/customApps.md (hunk 2 above)
Comment (savitharaghunathan) [reply docs/customApps.md]: fixed it. Thanks

File: docs/customApps.md (hunk 1 above)
Comment (savitharaghunathan) [reply docs/customApps.md]: updated link

**Review by jwmatthews — APPROVED** [review]
LGTM

**Pull request merged**